doc-content/enterprise-only/pages/building-custom-dashboard-widgets-creating-dashboard-proc.adoc

```diff
@@ -1,16 +1,18 @@
 [id='building-custom-dashboard-widgets-creating-dashboard-proc']
-= Creating a custom dashboard widget
+= {CENTRAL} custom dashboards
 
-In order to create custom dashboards, users must combine some of the features of the {CENTRAL} (like data sets, pages, navigation trees, and permissions) together and publish them. Dashboards are pages containing at least one reporting component.
+Dashboards are {CENTRAL} pages that contain at least one reporting component. Dashboards often contain data sets, navigation trees, and permissions.
+//Can a dashboard contain more that one page?
```

Comment:
Yes, a dashbaord always contains at least one page but it can contain as many pages as you wish.

Reply:
Thanks. Updated.